src/App.js

```diff
@@ -3,14 +3,20 @@ import { PageRouter } from "./PageRouter";
 import { Authenticator } from "@aws-amplify/ui-react";
 import { Amplify } from "aws-amplify";
 import authConfig from "./components/authentication/aws-export";
+import AlgoquantApiContext from "./ApiContext";
+import { AlgoQuant } from "../node_modules/algoquant/lib/AlgoQuant";
+
 import "./App.css";
 
 Amplify.configure({ ...authConfig, Analytics: { disabled: true } });
+const algoqant = new AlgoQuant();
```

Comment:
If you are creating a algoquant object in the `ApiContext.js` why are you creating it again here ? I don't think it should be done twice.

Reply:
took out the instance from APIContext.js , that one is needed to pass into the provider.